### DISCUSSION TITLE
Adding Granularity to Dropout Values

### DISCUSSION THREAD

**Discussion by BradKML** [keras-team/autokeras#1934] [Ideas]
Currently the dropout value follows `hyperparameters.Choice("dropout", [0.0, 0.25, 0.5], default=0.0)`
Proposal 1: Make the dropout more granular to `[0.0, 0.1, 0.2, 0.3, 0.4, 0.5]` or even more granular at 0.05 increments
Proposal 2: Usage of Keras Tuner to use `Float` with 0.1 or 0.05 steps, or with linear sampling https://keras.io/api/keras_tuner/hyperparameters/#float-method

Reference: https://github.com/keras-team/autokeras/blob/master/autokeras/blocks/heads.py https://github.com/keras-team/autokeras/blob/master/autokeras/blocks/basic.py
